packages/discovery-provider/src/queries/get_comments.py

```diff
@@ -70,6 +58,13 @@ def get_comment_replies(args, comment_id):
 
 def get_track_comments(args, track_id):
     offset, limit = format_offset(args), format_limit(args, COMMENT_THREADS_LIMIT)
+    sort_method = args.get("sort_method", "top")
+    if sort_method == "top":
+        order_by = desc(func.count(CommentReaction.comment_id))
```

Comment:
Nice, I think this is definitely the right way to start. But I do wonder whether we should weight using thread reactions and/or number of replies as well, gonna add that to our sync meeting topics tomorrow 🤔

Reply:
yes great idea!